src/logics/ArithLogic.cc

```diff
@@ -511,11 +518,11 @@ bool ArithLogic::isNumTerm(PTRef tr) const {
 PTRef ArithLogic::mkNeg(PTRef tr) {
     assert(!isNeg(tr)); // MB: The invariant now is that there is no "Minus" node
     SymRef symref = getSymRef(tr);
```

Comment:
As now all the if conditions are asking questions about `PTRef`, we no longer need to fetch `SymRef` here.

Reply:
Symref is used within some of the `if's` though...

Reply:
I'll move it inside the branches, but it will add 3 lines of code

Reply:
OK I see. then do not change the other if conditions. They can take keep taking the symref.